AwaitKit.podspec

```diff
@@ -3,10 +3,10 @@ Pod::Spec.new do |s|
   s.version          = '2.1.2'
   s.license          = 'MIT'
   s.summary          = 'The ES8 Async/Await control flow for Swift '
-  s.homepage         = 'https://github.com/yannickl/AwaitKit.git'
+  s.homepage         = 'https://github.com/arturgrigor/AwaitKit.git'
   s.social_media_url = 'https://twitter.com/yannickloriot'
   s.authors          = { 'Yannick Loriot' => 'contact@yannickloriot.com' }
-  s.source           = { :git => 'https://github.com/yannickl/AwaitKit.git', :tag => s.version }
+  s.source           = { :git => 'https://github.com/arturgrigor/AwaitKit.git', :tag => s.version }
```

Comment:
I've updated the link so that I could install the version of the pod from my fork, but I've chcked the "Allow edits from maintainers" option for the PR so you can restore them back.